README.md

```diff
@@ -8,9 +8,9 @@ Go to: [What is an RFC](#what-is-an-rfc) | [RFC Process](#rfc-process)
 <!--BEGIN_TABLE-->
 \#|Title|Owner|Status|
 ---|-----|-----|------|
-[9](https://github.com/aws-cloudformation/cfn-language-discussion/issues/9)|[Adding looping functionality in CFN Template](https://github.com/aws-cloudformation/cfn-language-discussion/pull/19)|[@MalikAtalla-AWS](https://github.com/MalikAtalla-AWS)|✍️ review
-[14](https://github.com/aws-cloudformation/cfn-language-discussion/issues/14)|[Convert Template Block to JSON String](https://github.com/aws-cloudformation/cfn-language-discussion/pull/15)|[@mluk-aws](https://github.com/mluk-aws)|✍️ review
-[11](https://github.com/aws-cloudformation/cfn-language-discussion/issues/11)|[Allow Intrinsic Functions and Pseudo-Parameter References in DeletionPolicy and UpdateReplacePolicy](https://github.com/aws-cloudformation/cfn-language-discussion/pull/21)|[@MalikAtalla-AWS](https://github.com/MalikAtalla-AWS)|✍️ review
+[9](https://github.com/aws-cloudformation/cfn-language-discussion/issues/9)|[Adding looping functionality in CFN Template](https://github.com/aws-cloudformation/cfn-language-discussion/pull/75)|[@MalikAtalla-AWS](https://github.com/MalikAtalla-AWS)|✍️ review
+[14](https://github.com/aws-cloudformation/cfn-language-discussion/issues/14)|[Convert Template Block to JSON String](https://github.com/aws-cloudformation/cfn-language-discussion/pull/76)|[@mluk-aws](https://github.com/mluk-aws)|✍️ review
+[11](https://github.com/aws-cloudformation/cfn-language-discussion/issues/11)|[Allow Intrinsic Functions and Pseudo-Parameter References in DeletionPolicy and UpdateReplacePolicy](https://github.com/aws-cloudformation/cfn-language-discussion/pull/74)|[@MalikAtalla-AWS](https://github.com/MalikAtalla-AWS)|✍️ approved
```

Comment:
Is this the right emoji for approved status? Do we have a key of emojis for each status somewhere?

Reply:
You're right. I didn't pay attention to that. I changed the status to "implementing" and used the same emoji CDK uses in their RFC repo.